cmd/podman/cleanup.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	cleanupFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "Cleans up all containers",
+		},
+		LatestFlag,
+	}
+	cleanupDescription = `
+   podman container cleanup
+
+   Cleans up mount points and network stacks on one or more containers from the host. The container name or ID can be used. This command is used internally when running containers, but can also be used if container cleanup has failed when a container exits.
+`
+	cleanupCommand = cli.Command{
+		Name:        "cleanup",
+		Usage:       "Cleanup network and mountpoints of one or more containers",
+		Description: cleanupDescription,
+		Flags:       cleanupFlags,
+		Action:      cleanupCmd,
+		ArgsUsage:   "CONTAINER-NAME [CONTAINER-NAME ...]",
+	}
+)
+
+func cleanupCmd(c *cli.Context) error {
+	if err := validateFlags(c, cleanupFlags); err != nil {
+		return err
+	}
+	runtime, err := libpodruntime.GetRuntime(c)
+	if err != nil {
+		return errors.Wrapf(err, "could not get runtime")
+	}
+	defer runtime.Shutdown(false)
+
+	args := c.Args()
+	if c.Bool("latest") && c.Bool("all") {
+		return errors.Errorf("--all and --latest cannot be used together")
+	}
+	if len(args) == 0 && !c.Bool("all") && !c.Bool("latest") {
```

Comment:
There are some cases you aren't covering here.  For example, if both `--latest` and an explicit container ID are given, you are using the latest container and ignoring the explicitly-given ID.  Perhaps something like:

```go
if c.Bool("all") {
  if c.Bool("lastest") {
    return errors.Errorf("--all and --latest cannot be used together")
  } else if len(args) != 0 {
    return errors.Errorf("--all and explicit container IDs cannot be used together")
  }
} else if c.Bool("latest") {
  if len(args) != 0 {
    return errors.Errorf("--latest and explicit container IDs cannot be used together")
  }
} else if len(args) == 0 {
  return errors.Errorf("you must provide at least one container name or id")
}
```

Also, there's no formatting going on here, so [`Errorf` from the GitHub package][1] seems like overkill.  Can we use [the stdlib's `errors.New`][2]?

[1]: https://godoc.org/github.com/pkg/errors#Errorf
[2]: https://golang.org/pkg/errors/#New